### PR TITLE
UI Component - Alert

### PR DESCRIPTION
**Pull request by shreeyash07**

### CODE REVIEW

File: src/root/views/GoUI/components/Alert/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      align-items: initial;
+    }
+
+    p {
+      margin: 0;
+    }
+
+    strong {
+      font-weight: var(--go-ui-font-weight-bold);
+    }
```
Comment (samshara) [inline src/root/views/GoUI/components/Alert/styles.module.scss:38]: any reason we are using these selectors here? can't we style using class selectors?

File: src/root/views/GoUI/components/Alert/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  .actions {
+    display: flex;
+    justify-content: flex-end;
+    padding: $spacing * 0.5;
```
Comment (samshara) [inline src/root/views/GoUI/components/Alert/styles.module.scss:44]: use CSS variables as defined in the root view.

File: src/root/views/GoUI/components/Alert/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    font-weight: var(--go-ui-font-weight-bold);
+
+    .copy-debug-message-button {
+      padding: $spacing * 0.5;
```
Comment (samshara) [inline src/root/views/GoUI/components/Alert/styles.module.scss:48]: similar to above

File: src/root/views/GoUI/styles.module.scss [hunk 4]
```diff
@@ -142,6 +142,10 @@
     --go-ui-color-warning: var(--go-ui-color-yellow);
     --go-ui-color-negative: var(--go-ui-color-red);
 
+    --go-ui-color-success: #009896;
+    --go-ui-color-warning: #f39c12;
+    --go-ui-color-danger: var(--go-ui-color-red-90);
```
Comment (samshara) [inline src/root/views/GoUI/styles.module.scss:147]: let's use semantic colors defined already.

File: src/root/views/GoUI/components/Alert/styles.module.scss (hunk 1 above)
Comment (shreeyash07) [reply src/root/views/GoUI/components/Alert/styles.module.scss]: I have changed these css

**Review by samshara — APPROVED** [review]

**Pull request merged**